api/discordsession.go

```diff
@@ -1,11 +1,11 @@
 package api
 
-import "github.com/aetimmes/discordgo"
+import "github.com/bwmarrin/discordgo"
 
 // DiscordSession is an interface for interacting with Discord within a session
 // message handler.
 type DiscordSession interface {
-	ChannelMessageSend(channel, message string) (*discordgo.Message, error)
-	Channel(channelID string) (*discordgo.Channel, error)
-	User(userID string) (*discordgo.User, error)
+	ChannelMessageSend(channelID string, content string, options ...discordgo.RequestOption) (*discordgo.Message, error)
```

Comment:
In Go it's convention to only write the type once for adjacent params. So the signature would be..

```
channelID, content string, options ...discordgo.RequestOption
```

Reply:
Ack, will update